### PR TITLE
docs: update GA status

### PR DESCRIPTION
**Pull request by jannyHou**
connect to https://github.com/strongloop/loopback-next/issues/1744

PLEASE DO NOT MERGE IT UNTIL GA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,20 +13,18 @@ integrations.
...
+The documentation website is https://github.com/strongloop/loopback-next.
 
 Learn about the latest and greatest
 [features and technologies in LoopBack 4](https://loopback.io/doc/en/lb4/Crafting-LoopBack-4.html)
 by using it for your next project. Start by having a look at
 [Getting Started](https://loopback.io/doc/en/lb4/Getting-started.html).
 
-**NOTE:** _It is not recommended for production use yet._ For production, use
-[LoopBack 3](https://loopback.io/doc/en/lb3/index.html).
+Check the [API documentation](https://apidocs.strongloop.com/@loopback%2fdocs/apidocs.html) for all the API usages in each package.
```
Comment (dhmlau) [inline README.md]: actually we want to promote `http://apidocs.loopback.io/...` instead of `http://apidocs.strongloop.com/...`

File: README.md [hunk 2]
```diff
@@ -13,20 +13,18 @@ integrations.
...
 - No maintenance of generated code
 
-## Status: Developer Preview #3
+## Status: General Availability
 
-LoopBack 4 is a work in progress, the public API is frequently changed in
-backward incompatible ways. See
-[Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
-for more details.
+LoopBack 4 has entered the GA phase.
```
Comment (dhmlau) [inline README.md]: don't think we need this line, or have something like "LoopBack 4 has GA'ed"

File: README.md [hunk 3]
```diff
@@ -13,20 +13,18 @@ integrations.
...
-## Status: Developer Preview #3
+## Status: General Availability
 
-LoopBack 4 is a work in progress, the public API is frequently changed in
-backward incompatible ways. See
-[Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
-for more details.
+LoopBack 4 has entered the GA phase. 
+
+The documentation website is https://github.com/strongloop/loopback-next.
```
Comment (dhmlau) [inline README.md]: url: https://loopback.io/doc/en/lb4/

**Comment by bajtos** [comment]
Can we add a link to Module LTS policy please, explain which LoopBack versions are Current/ActiveLTS/MaintenaceLTS and perhaps include a link to https://github.com/strongloop/loopback-next for people looking for an LTS version.

File: README.md (hunk 3 above)
Comment (jannyHou) [reply README.md]: oops! good catch lol

**Comment by dhmlau** [comment]
@bajtos , good point for the module LTS policy.  
I think it's something like https://github.com/strongloop/loopback#module-long-term-support-policy
Info can be found here: https://loopback.io/doc/en/contrib/Long-term-support.html

File: README.md [hunk 4]
```diff
@@ -13,20 +13,20 @@ integrations.
...
 - No maintenance of generated code
 
-## Status: Developer Preview #3
+## Status: General Availability
 
-LoopBack 4 is a work in progress, the public API is frequently changed in
-backward incompatible ways. See
-[Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
-for more details.
+LoopBack 4 has GA'ed.
```
Comment (dhmlau) [inline README.md]: Using @bajtos' suggestion on the v4.loopback.io PR, perhaps we can use the same:

LoopBack 4 GA (General Availability) has been released in October 2018, read more in [the announcement post](http://strongloop.com/strongblog/loopback-4-ga). 

Thanks!

**Comment by dreamdevil00** [comment]
Congratrulations!

**Pull request merged**